### PR TITLE
Issue 660 Refactor HistoryAPI isConfirmed

### PR DESCRIPTION
**Pull request by iammatis**

### CODE REVIEW

**Comment by iammatis** [comment]
`Model/TransactionTest` failing test is fixed in #20

File: src/Sdk/HistoryApiInterface.php [hunk 1]
```diff
@@ -59,4 +59,17 @@ public function listOperationsRelative(ChainObject $accountId, int $start = 0, i
...
     public function findAllOperations(ChainObject $accountId, array $assets = [], ChainObject $recipientAccount = null, string $fromBlock = '0', string $toBlock = '0', string $startOffset = '0', int $limit = 100): array;
 
+    /**
+     * Verifies if block in that transaction was processed to is irreversible.
+     * NOTE: Unverified blocks still can be reversed.
+     *
+     * NOTICE:
+     * Transaction object with id in form '1.7.X' can be fetched from AccountApi->getAccountHistory() method.
+     *
+     * @param ChainObject $transactionId
```
Comment (petervanderkadecent) [inline src/Sdk/HistoryApiInterface.php]: This should be operation id, operation history object to be specific.

File: src/Sdk/HistoryApiInterface.php [hunk 2]
```diff
@@ -66,10 +68,10 @@ public function findAllOperations(ChainObject $accountId, array $assets = [], Ch
      * NOTICE:
      * Transaction object with id in form '1.7.X' can be fetched from AccountApi->getAccountHistory() method.
```
Comment (petervanderkadecent) [inline src/Sdk/HistoryApiInterface.php]: I think it is called history object to be consistent.

Comment (iammatis) [reply src/Sdk/HistoryApiInterface.php]: Renamed to `History object...`

**Pull request merged**